proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -71,10 +71,23 @@ message GetResponse {
 
 message GetLogRequest {
   string session_id = 1 [(buf.validate.field).string.min_len = 1];
+  int32 page_size = 20;
```

Comment:
didn't we say we'll use PaginationOptions or some such?

Reply:
Nope

We said we to design a generic thing and do it once all over

Currently there is no more requirement for pagination, only two 

Third request is the time to generalize